src/asset_manager/view/FileUploader.js

```diff
@@ -32,7 +32,7 @@ module.exports = Backbone.View.extend(
 
       if (uploadFile) {
         this.uploadFile = uploadFile.bind(this);
-      } else if (c.embedAsBase64) {
+      } else if (!c.upload && c.embedAsBase64) {
```

Comment:
@artf perhaps this way it would be satisfactory. `embedAsBase64` gets attached as long as `uploadFile` or `upload` is not configured.

This way there wouldn't be a non-functioning drop area by default.

Reply:
Good idea, I like it 👍